### PR TITLE
TNO-2091/93: New nav bar and layout

### PR DESCRIPTION
**Pull request by jdtoombs**
- change grid layout to have header, sidebar, and main content 
- update theme
- new navbar
- css enhancements 

image

image

### CODE REVIEW

File: app/subscriber/src/components/navbar/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  groupName: string;
+  sortOrder: number;
+}
+
+export interface INavbarProps {
+  options?: INavbarOptions[];
+}
+
+export const Navbar: React.FC<INavbarProps> = ({ options }) => {
+  const [expanded, setExpanded] = React.useState(true);
```
Comment (Fosol) [inline app/subscriber/src/components/navbar/Navbar.tsx:21]: Keep your `use*` dependencies first.
Then `useState`
Then variables
Then `useEffect`

It makes it easier to read and consistent with all components

File: app/subscriber/src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          return (
+            <Row className="option" key={index} onClick={() => navigate(option.path)}>
+              {option.icon}
+              <Show visible={expanded}>{option.label}</Show>
+            </Row>
+          );
+        })}
+
+        {Object.keys(result).map((key) => {
+          console.log(key);
```
Comment (Fosol) [inline app/subscriber/src/components/navbar/Navbar.tsx:41]: remove

File: app/subscriber/src/components/navbar/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
```
Comment (jdtoombs) [inline app/subscriber/src/components/navbar/Navbar.tsx:1]: will add some docs

File: app/subscriber/src/components/navbar/Navbar.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/subscriber/src/components/navbar/Navbar.tsx:23]: Darn it tried to sneak that in before you saw, updated now lol

File: app/subscriber/src/components/navbar/Navbar.tsx (hunk 2 above)
Comment (jdtoombs) [reply app/subscriber/src/components/navbar/Navbar.tsx]: ready for review by the way! @Fosol

**Review by Fosol — APPROVED** [review]

**Pull request merged**